modulemd/v2/include/modulemd-2.0/modulemd-module-stream.h

```diff
@@ -352,11 +380,28 @@ modulemd_module_stream_get_context (ModulemdModuleStream *self);
  * module stream. NULL if module name or stream stream is unknown.
  *
  * Since: 2.0
+ * Deprecated: 2.2
  */
+MMD_DEPRECATED_FOR (modulemd_module_stream_get_NSVCA_as_string)
 gchar *
 modulemd_module_stream_get_nsvc_as_string (ModulemdModuleStream *self);
 
 
+/**
+ * modulemd_module_stream_get_NSVCA_as_string: (rename-to modulemd_module_stream_get_NSVCA)
+ * @self: (in): This #ModulemdModuleStream.
+ *
+ * Returns: (transfer full): The NSVCA of this module stream. The returned
+ * format is described
+ * [here](https://docs.fedoraproject.org/en-US/modularity/architecture/consuming/naming-policy/#_forms).
```

Comment:
Will this reference work when generating the documentation? I don't recall if `gtk-doc` actually handles Markdown.

Reply:
https://developer.gnome.org/gtk-doc-manual/stable/documenting_syntax.html.en

> As of version 1.20, the preferred way is to use a subset of the basic text formatting syntax called Markdown. On older GTK-Doc versions any documentation that includes Markdown will be rendered as is.

So this should work fine on any platform we're likely to care about.